sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -2359,6 +2360,77 @@ def test_pretty_Lambda():
     assert upretty(expr) == ucode_str
 
 
+def test_pretty_TransferFunction():
+    tf1 = TransferFunction(s - 1, s + 1, s)
+    assert upretty(tf1) == u"s - 1\n─────\ns + 1"
+    tf2 = TransferFunction(2*s + 1, 3 - p, s)
+    assert upretty(tf2) == u"2⋅s + 1\n───────\n 3 - p "
+    tf3 = TransferFunction(p, p + 1, p)
+    assert upretty(tf3) == u"  p  \n─────\np + 1"
+
+
+def test_pretty_Series():
+    tf1 = TransferFunction(x + y, x - 2*y, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    tf3 = TransferFunction(x**2 + y, y - x, y)
+    expected1 = \
+u("""\
+         ⎛ 2    ⎞ \n\
+ (x + y)⋅⎝x  + y⎠ \n\
+──────────────────\n\
+(-x + y)⋅(x - 2⋅y)\
+""")
+    expected2 = \
+u("""\
+(-x - y)⋅(-x + y)\n\
+─────────────────\n\
+(x - 2⋅y)⋅(x + y)\
+""")
+    assert upretty(Series(tf1, tf3)) == expected1
+    assert upretty(Series(-tf2, -tf1)) == expected2
+
+
+def test_pretty_Parallel():
+    tf1 = TransferFunction(x + y, x - 2*y, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    expected1 = \
+u("""\
+                           2\n\
+(x - 2⋅y)⋅(x - y) + (x + y) \n\
+────────────────────────────\n\
+     (x - 2⋅y)⋅(x + y)      \
+""")
+    expected2 = \
+u("""\
+(-x - y)⋅(x + y) + (-x + y)⋅(x - 2⋅y)\n\
+─────────────────────────────────────\n\
+          (x - 2⋅y)⋅(x + y)          \
+""")
+    assert upretty(Parallel(tf1, tf2)) == expected1
+    assert upretty(Parallel(-tf2, -tf1)) == expected2
```

Comment:
I think these should print like:

```
x + y     x - y
------  + -------
x - 2y   x + y
```

Reply:
Initially, I thought of doing the same thing. But consider a transfer function operation like this: `(tf1 + tf2)*(tf3 + tf4)` how to add brackets in this?

Reply:
Something like this:
```
(x + y      x - y  )  (x + y      x - y ) 
|------  + ------- |  |------  + -------|
(x - 2y    x + y   )  (x + y      x - y )
```

Reply:
There should be similar examples when printing expressions of fractions.

Reply:
How about this?

``` python
 [x + y      x - y  ]   [x + y      x - y ] 
 [------  + ------- ] * [------  + -------]
 [x - 2y    x + y   ]   [x + y      x - y ]
```

Reply:
There are functions in the printing code to add parentheses